spec/02-integration/04-admin_api/12-plugins-conf_spec.lua

```diff
@@ -31,6 +35,27 @@ describe("Plugins conf property" , function()
       assert.equal(tablex.size(bundled_plugins),
                    tablex.size(json.plugins.available_on_server))
     end)
+    it("expect all plugins are in bundled", function()
+      local res = assert(client:send {
+        method = "GET",
+        path = "/",
+      })
+      local body = assert.res_status(200 , res)
+      local json = assert(cjson.decode(body))
+      local bundled_plugins_list = json.plugins.available_on_server
+      local rocks_installed_plugins, err = utils.execute("ls -1 $(luarocks show kong | grep 'kong.plugins' | grep -E '/.+/kong/plugins/' -o | uniq)")
```

Comment:
maybe we **could** semplify this a bit (I just tested it quickly, we should confirm it works), up to you though
```suggestion
            local rocks_installed_plugins, err = utils.execute("luarocks show kong | grep -oP 'kong\\.plugins\\.\\K([\\w-]+)' | uniq")
```